extensions/Worker.Extensions.SignalRService/SignalREndpointsAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    public sealed class SignalREndpointsAttribute : InputBindingAttribute
```

Comment:
Will this work as a class attribute when applied to this model?
https://docs.microsoft.com/en-us/azure/azure-signalr/signalr-concept-serverless-development-config#negotiate-experience-in-class-based-model

Reply:
No.